packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -157,6 +157,7 @@
     let bindings = []
     let loopBlockCount = 0
     const addBinding = (name, value, icon, idx, isLoopBlock, bindingName) => {
+      if (!name) return
```

Comment:
When will this happen? This is running for each key on an object

Reply:
![image](https://github.com/Budibase/budibase/assets/5665926/1d6e1ebe-70cc-40c5-8601-9002f204deea)

The key can actually be null for this pair so it's just a safeguard against that.